modules/ROOT/pages/upgrade.adoc

```diff
@@ -58,44 +55,26 @@ The `sudo ./upgrade` command updates the platform to the current version and res
 ----
 -> initializing upload
 -> uploading update package
-importing anypoint:2.0.0
+importing anypoint:2.1.0
 Thu Aug 23 09:36:30 UTC	Synchronizing application with Docker registry x.x.x.xxx:5000
 Thu Aug 23 09:44:27 UTC	Synchronizing application with Docker registry x.x.x.xxx:5000
 Thu Aug 23 09:51:29 UTC	Synchronizing application with Docker registry x.x.x.xxx:5000
 Thu Aug 23 09:58:23 UTC	Application has been uploaded
-updating anypoint from 1.7.1 to 2.0.0-xxxxxxxxxxxx
+updating anypoint from 2.0.4 to 2.1.0-xxxxxxxxxxxx
 update operation (d077277f-3c69-4336-82bf-bd8b428ae202) has been started
 the cluster is updating in background
 ----
 
-. When the upgrade finishes, verify that the migration was successful by performing one of the following options:
-+
-* Run the following command from the node where you performed the upgrade:
+. When the upgrade finishes, verify the migration was successful by checking that all pods are running:
 +
 ----
 $ sudo gravity enter
-$ kubectl get pods
+$ kubectl get pods --all-namespaces
 ----
 
-. Log in to the Ops Center at the following URL:
-+
-----
-https://{platform-host}:9500
-----
-+
-[WARNING]
-If you are upgrading from Anypoint Platform PCE 1.7.x to 2.0.x, ensure that you configure your NSF servers described in the following step before accessing Anypoint Platform PCE. In this case, failure to configure NFS before accessing the platform can result in data loss.
+. Open the new `8083` port on your load balancer and map it to the `30883` port on your servers. Refer to the 2.1 xref:prereq-hardware.adoc[Hardware Prerequisities] section for more information. 
 
-. When upgrading from from Anypoint Platform PCE 1.7.x to 2.0.x, configure the required NFS server.
-+
-.. From Anypoint Platform, select *Access Management*.
-.. Select *NFS Settings*.
-.. Enter the domain name of your NFS server.
-.. Enter the path to your NFS server.
-.. Select *Save*.
-.. Select *Test NFS Server* to test that your NFS server is configured correctly.
-
-. Unless you are updating Anypoint Platform PCE from 2.0.x to 2.1.x, configure the following items:
+. Finally, make sure the following items are still correctly configured after the update:
```

Comment:
I know these seem trivial, but I am thinking of a global audience, which can have trouble with colocations (see style ref):
```suggestion
. Finally, ensure that the following items are still correctly configured after the update:
```